test/shared/src/main/scala/zio/test/package.scala

```diff
@@ -27,12 +27,13 @@ package zio
  *
  * {{{
```

Comment:
Should we introduce something that type-checks Scaladoc or move these examples into the microsite?

Reply:
For sure, but let's wait till API stabilizes a bit, probably another week.